lib/src/base-styles/_nx-btn.scss

```diff
@@ -6,48 +6,67 @@
  */
 @use '../scss-shared/nx-container-helpers';
 @use '../scss-shared/nx-text-helpers';
+@use '../scss-shared/nx-dark-mode-helpers';
 
-%button-focus-outline {
-  outline-offset: -5px;
-  outline: 2px solid;
+$transparent-button-outline-offset: calc(0px - var(--nx-border-width-button) - 4px);
```

Comment:
> To make it a local variable, inaccessible outside this file:

Actually, this cleaned up even better than I had realized it would, and this variable is only used in one place, which means it really doesn't need to be a variable at all

Reply:
> > To make it a local variable, inaccessible outside this file:
> 
> Actually, this cleaned up even better than I had realized it would, and this variable is only used in one place, which means it really doesn't need to be a variable at all

Removed here: [deac080](https://github.com/sonatype/sonatype-react-shared-components/pull/1038/commits/deac0808835df98a9e5aada5b9e11842e073737b)